plasmapy/atomic/atomic.py

```diff
@@ -696,6 +696,9 @@ def isotope_mass(argument, mass_numb=None):
 
     argument, charge_state = _extract_charge_state(argument)
 
+    if _is_proton(argument) or _is_antiproton(argument):
+        return const.m_p
```

Comment:
Test proton and antiproton masses in `isotope_mass`.  Make sure that special cases are documented.